test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -33,7 +33,7 @@ apply plugin: 'opensearch.java'
 group = 'hdfs'
 
 versions << [
-  'jetty': '9.4.51.v20230217'
+  'jetty': '12.0.1'
```

Comment:
@DarshitChanpura this is a stretch probably, AFAIK 9.x and 12.x are not compatible

Reply:
We may need to use 9.4.52.v20230823, it fixes GMS-2023-1857 as well, as per release notes https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.52.v20230823

Reply:
hmm. I updated them separately.

Reply:
Why? Should be just

```
'jetty': '9.4.52.v20230823'
```

I think

Reply:
done. I went with a basic thinking to upgrade both to their latest versions but that seems to cause:
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/eclipse/jetty/util/MultiException
```

i have synced both to `9.4.52.v20230823`